auth/src/main/java/com/vimeo/networking2/internal/AuthService.kt

```diff
@@ -146,7 +148,8 @@ internal interface AuthService {
     ): VimeoCall<VimeoAccount>
 
     /**
-     * Used to log into Vimeo using a Facebook authorization token.
+     * Used to log into Vimeo using a Facebook authorization token. If an account does not exist, this request will
+     * fail. For this reason, [joinWithFacebook] is preferred since it also supports login.
```

Comment:
I want to keep them around as documentation of their existence and how they can be used in contrast with the `joinWith...` functions. Since `AuthService` is internal, this will not be confusing to the user, only us as maintainers. I added some more docs to these functions to explain that they are intentionally unused and added suppression of the unused warnings.